### DISCUSSION TITLE
SPFx 1.16.1 Dark mode detection

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [SharePoint/sp-dev-docs#8665] [Q&A]
I want to implement support to Dark mode but it does not seam to figure out that the browser (and Windows) is in dark mode.

The line :`this._isDarkTheme = !!currentTheme.isInverted;`  the value currentTheme.isInverted is always false.

Is there something else I did not do?

Thank you

**Comment by AJIXuMuK** [comment]
@Ofer-Gal - could you please provide your scenario?
Is it for Teams Dark mode? Or SharePoint "dark" themes?

**Comment by Ofer-Gal** [comment]
This is running code in /_layouts/workbench.aspx  in chrome and edge set to be in Dark Mode.
Could it be that it is only targeted to a site that has a "Dark" Theme? I did not test that your scenario.
Thank you
